shared/LICENSE.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should license.json still be in shared?

Reply:
I see no reason why not!, ill move it.

Reply:
actually this was duplicated, i will delete this extra.